src/App.tsx

```diff
@@ -17,21 +17,17 @@ function App() {
           <Route
             path="/home"
             element={
-              <PrivateRoute
-                children={<Home />}
-                value={"/signup"}
-                loggedIn={loggedIn}
-              />
+              <PrivateRoute value={"/signup"} loggedIn={loggedIn}>
```

Comment:
Instead of passing loggedIn prop to every `<PrivateRoute />` render, access its value inside PrivateRoute component using redux hooks.